message-me/db/migrate/20210721103106_create_messages.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMessages < ActiveRecord::Migration[6.1]
+  def change
+    create_table :messages do |t|
+      t.string :body
```

Comment:
Why `string`?

Reply:
should be `text` because it can hold more characters compared to string. Which is more suitable in our case.